package.json

```diff
@@ -4,156 +4,64 @@
   "author": "Baruch Rothkoff <baruchiro@gmail.com>",
   "description": "An Electron-vue project for israeli-bank-scrapers",
   "license": "MIT",
-  "main": "./dist/electron/main.js",
+  "private": true,
```

Comment:
OK

Reply:
@Arielgordon123 I think we need to mark as private, to avoid publishing as package...